pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 1.4.0
+2020-09-25
+
+* Updated GATK docker image for all tasks from [GATK 4.1.1.0](https://github.com/broadinstitute/gatk/releases/tag/4.1.1.0) to [GATK 4.1.8.0](https://github.com/broadinstitute/gatk/releases/tag/4.1.8.0).
+    * Numerous bug fixes and improvements, including better VQSR reliability for allele-specific filtering.
+* Updated picard-cloud docker image from a custom branch to [Picard 2.23.0](https://github.com/broadinstitute/picard/releases/tag/2.23.0)
```

Comment:
Did we update a picard docker here? I only saw GATK, but I may have missed it

Reply:
You are correct, we're not using picard in any tasks here.  I've removed this from the changelogs.